io/EptReader.hpp

```diff
@@ -71,6 +74,14 @@ class PDAL_DLL EptReader : public Reader
     std::string getName() const override;
 
 private:
+    uint64_t m_nodeId = 1;
```

Comment:
Could you please group those member variables at the bottom of the class definition with the otehr member variables?